Update _index.md on the runtime_id_enabled (#29782)
DataDog/documentation: content/en/tracing/metrics/runtime_metrics/_index.md

: **Default**: `true` for Java, `false` for all other languages <br>
**Description**: Enables the collection of runtime metrics. Metrics are sent to the Datadog agent, as configured for the instrumented application.

`DD_RUNTIME_METRICS_RUNTIME_ID_ENABLED`

> : **Default**: `true` for Java, `false` for Node.js, Ruby and Python. Does not exist for .NET and Go, the `runtime_id` is always reported <br>

Reviewer: ```suggestion
: **Default**: `true` for Java, `false` for Node.js, Ruby, and Python. Does not exist for .NET and Go; the `runtime_id` is always reported <br>
```

> : **Default**: `true` for Java, `false` for Node.js, Ruby, and Python. Does not exist for .NET and Go; the `runtime_id` is always reported <br>

Reviewer 2: ```suggestion
: **Default**: `true` for Java, `false` for Node.js, Ruby, and Python. Does not exist for .NET and Go; the `runtime_id` is always reported. <br>
```

: **Default**: `true` for Java, `false` for all other languages <br>
**Description**: Enables the collection of runtime metrics. Metrics are sent to the Datadog agent, as configured for the instrumented application.

`DD_RUNTIME_METRICS_RUNTIME_ID_ENABLED`

> : **Default**: `true` for Java, `false` for Node.js, Ruby and Python. Does not exist for .NET and Go, runtime-id is always reported <br>

Author: @bm1549 changed to this as .net and go just don't have the env var and it's not configurable, so itwont work if someone tries to deactivate it for these languages. Let me know if you think it's too complicated and we should just say instead:

> true for Java, .NET and Go, false for all other languages.

: **Default**: `true` for Java, `false` for all other languages <br>
**Description**: Enables the collection of runtime metrics. Metrics are sent to the Datadog agent, as configured for the instrumented application.

`DD_RUNTIME_ID_ENABLED`

> : **Default**: `false` for NodeJs, Ruby and Python languages <br>

Reviewer 3: ```suggestion
: **Default**: `false` for Node.js, Ruby and Python languages <br>
```

> : **Default**: `false` for Node.js, Ruby and Python languages <br>

Author: see [below ](https://github.com/DataDog/documentation/pull/29782/#discussion_r2163681695)

> **Description**: Provides a runtime_id tag along with every metric. The runtime_id represents the application's process identifier and allows you to directly correlate runtime metrics with individual running applications.

Reviewer 3: ```suggestion
**Description**: Provides a `runtime_id` tag along with every metric. The `runtime_id` represents the application's process identifier and allows you to directly correlate runtime metrics with individual running applications.
```

> **Description**: Enables enhanced runtime metrics, providing a `runtime_id` tag along with every metric. The `runtime_id` represents the application's process identifier and allows you to directly correlate runtime metrics with individual running applications.

Author: @bm1549 changed to this as .net and go just don't have the env var and it's not configurable, so itwont work if someone tries to deactivate it for these languages. Let me know if you think it's too complicated and we should just say instead:
> `true` for Java, .NET and Go, `false` for all other languages.
